### PR TITLE
TDD of AudioConverter class

### PR DESCRIPTION
**Pull request by Enrique-C**

### CODE REVIEW

File: src/main/java/com/jalasoft/jfc/model/audio/AudioConverter.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ * @author Enrique Carrizales.
+ */
+public class AudioConverter implements IConverter {
+
+    /**
+     * Changes an Audio format to another one.
+     * @param param it receives file params.
+     * @return FileResponse object.
+     * @throws ConvertException when the conversion failed.
+     * @throws CommandValueException when is a invalid command.
```
Comment (JuanCarlosMartinez4) [inline src/main/java/com/jalasoft/jfc/model/audio/AudioConverter.java]: improve this comment

File: src/main/java/com/jalasoft/jfc/model/audio/AudioConverter.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+     * @throws IOException when is a wrong input.
```
Comment (JuanCarlosMartinez4) [inline src/main/java/com/jalasoft/jfc/model/audio/AudioConverter.java]: when there is...

File: src/main/java/com/jalasoft/jfc/model/audio/AudioConverter.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+     * @throws ZipJfcException when zip process is wrong.
```
Comment (JuanCarlosMartinez4) [inline src/main/java/com/jalasoft/jfc/model/audio/AudioConverter.java]: when zip process was wrong

File: src/test/java/com/jalasoft/jfc/model/audio/AudioConverterTest.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        boolean expected = zipFile.exists() && zipFile.getTotalSpace() > EMPTY_BYTES;
+
+        FolderRemover.removeFolder(zipFile.getPath());
+
+        assertTrue(expected);
+    }
+
+    private AudioParam generateOnlyAudioWAV() {
+        PathJfc pathJfc = new PathJfc();
+
```
Comment (JuanCarlosMartinez4) [inline src/test/java/com/jalasoft/jfc/model/audio/AudioConverterTest.java:51]: delete this line

Comment (Enrique-C) [reply src/test/java/com/jalasoft/jfc/model/audio/AudioConverterTest.java:51]: I add this code line to separate it the other value

**Review by JuanCarlosMartinez4 — APPROVED** [review]

**Pull request merged**